roles/dnsmasq/tasks/configure.yml

```diff
@@ -60,6 +60,12 @@
     src: "cifmw-dnsmasq.conf.j2"
     validate: "/usr/sbin/dnsmasq -C %s --test"
 
+- name: Render listener configuration
```

Comment:
so this means we're expected to know everything before we start to deploy dnsmasq? It may get slightly ugly pretty fast with the way we'll call things:
- deploy dnsmasq basics
- create libvirt networks (and add configuration snippets to dnsmasq for the DHCP)
- get to devscripts parts, where we have to inject yet other snippets, some related to DNS, listeners and so on

We therefore want to set target file names to ensure we can add content "on the fly".